app/models/shopify/import/client.rb

```diff
@@ -0,0 +1,35 @@
+require 'singleton'
+
+module Shopify
+  module Import
+    class Client
+      include Singleton
+      attr_reader :site
+
+      def get_connection(api_key: nil, password: nil, shop_name: nil)
+        @api_key = api_key || Spree::Config[:shopify_api_key]
+        @password = password || Spree::Config[:shopify_password]
+        @shop_name = shop_name || Spree::Config[:shopify_shop_name]
```

Comment:
The idea was to use `shop_domain` that will **include** ".myshopify.com", so the value of `shop_domain` would be something like:

```ruby
shop_domain = 'example.myshopify.com'
```

Currently, the only thing that changed is a variable name. Which is now incorrectly called `shop_domain`, but represents only myshopify subdomain.